stories/Grid.story.jsx

```diff
@@ -377,13 +377,11 @@ storiesOf("Grid", module)
     ))
```

Comment:
please dont let this bit get you confused.
I just reordered the stories so that the responsive story goes after the text-align-right story.